src/fides/api/alembic/migrations/versions/14acee6f5459_translation_data_clear_experience_configs.py

```diff
@@ -0,0 +1,565 @@
+    US_MODAL = "pri-76b8-cc52-11ee-9eaa-8ef97a0-over-config"
+    US_PRIVACY_CENTER = "pri-27d4-cc53-11ee-9eaa-8ef97a04-pri-config"
+
+
+class ComponentType(Enum):
+    PrivacyCenter = "privacy_center"
+    Overlay = "overlay"
+    TCFOverlay = "tcf_overlay"
+
+
+EEA_MAPPING = {
+    ComponentType.PrivacyCenter: DefaultExperienceConfigTypes.EEA_PRIVACY_CENTER,
+    ComponentType.Overlay: DefaultExperienceConfigTypes.EEA_MODAL_AND_BANNER,
+    ComponentType.TCFOverlay: DefaultExperienceConfigTypes.EEA_TCF_OVERLAY,
+}
+US_MAPPING = {
+    ComponentType.PrivacyCenter: DefaultExperienceConfigTypes.US_PRIVACY_CENTER,
+    ComponentType.Overlay: DefaultExperienceConfigTypes.US_MODAL,
+}
+
+NOTICES_TO_EXPERIENCE_CONFIG = {
+    "marketing": EEA_MAPPING,
+    "functional": EEA_MAPPING,
+    "essential": EEA_MAPPING,
+    "analytics": EEA_MAPPING,
+    "data_sales_and_sharing": US_MAPPING,
+}
+
+
+def load_experience_config_from_files(file_paths):
+    """
+    Attempts loading the experience config from a set of given file paths until it finds a valid yaml
+    """
+    for path in file_paths:
+        try:
+            with open(path, "r") as f:
+                return yaml.safe_load(f).get("privacy_experience_configs")
+        except FileNotFoundError:
+            # If the file is not found in the current path, try the next one
+            continue
+
+    return None
+
+
+def load_default_experience_configs():
+    """
+    Loads default experience config definitions from yml file.
+
+    Returns two dicts in a tuple:
+        - first dict holds "new" experience config records that will be used to reconcile existing data
+        - second dict holds the "raw" experience config records based on the definitions on disk. these will be used as a reference for e.g. template creation.
+    """
+    _raw_experience_config_map = (
+        {}
+    )  # this will just hold config templates exactly as they are in the 'oob' file
+    _reconciled_experience_config_map = (
+        {}
+    )  # will hold experience configs 'reconciled' between existing data and 'oob' template data
+
+    # this is a bit of a hack! but the path to the file depends on whether we're executing the migration
+    # as part of app startup or as part of a 'manual' migration attempt
+    experience_configs = load_experience_config_from_files(
+        [
+            "privacy_experience_config_defaults.yml",
+            "src/fides/api/alembic/privacy_experience_config_defaults.yml",
+        ]
+    )
+
+    for experience_config in experience_configs:
+        _raw_experience_config_map[
+            DefaultExperienceConfigTypes(experience_config["id"])
+        ] = experience_config.copy()
+        experience_config["regions"] = (
+            set()
+        )  # remove the configs regions, since we will populate based on existing notices
+        experience_config["privacy_notices"] = set()  # initialize the notices field
```

Comment:
yup a good thing to point out. as mentioned in my other comment, the underlying assumption for the TCF experience was that it would have no changes made to it so it'd fall back to the 'default' template logic, but that may not be a good assumption...

Reply:
yup not a good assumption, have confirmed we have at least one customer who has edited the TCF overlay experience config 👍

Reply:
this should be handled now, confirmed with some manual testing 👍